bin/main.rb

```diff
@@ -1,49 +1,55 @@
 # !/usr/bin/env ruby
 
-# require_relative '../lib/tic_tac_toe'
-# require_relative "../lib/player"
+# rubocop:disable Metrics/AbcSize, Metrics/MethodLength, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
```

Comment:
- [ ] [OPTIONAL] You should not overuse RuboCop disable cops comments. only the complexity ones are allowed to be ignored such as `Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity`.

Reply:
@rukundoeric Thank you for the review.

Noted. We will remove it after refactoring further.

If it is approved can we go ahead and merge?